NashvilleHistory/www/js/controllers/AllPlacesCtrl.js

```diff
@@ -1,14 +1,26 @@
 'use strict';
 
-app.controller('AllPlacesCtrl', function($scope, $state, $q, AllPlacesFact){
+app.controller('AllPlacesCtrl', function($scope, $state, $q, AllPlacesFact, BookmarkFact, AuthFact){
 
   let AllPlaces;
   let HistoricalMarkers;
   let ArtMarkers;
   let CivilWarMarkers;
+  let AllMarkers;
   $scope.artFilter = false;
   $scope.historicalFilter = false;
   $scope.civilWarFilter = false;
+  // Auto-Login Handler
```

Comment:
We can keep this logic on the parent scope in `AppCtrl.js` so it persists across all pages. You can reference `$parent.loggedInUser` instead wherever necessary